os2borgerpc_client/os2borgerpc/client/config.py

```diff
@@ -4,8 +4,7 @@
 import sys
 import stat
 
-DEFAULT_CONFIG_FILES = [
-        "/etc/os2borgerpc/os2borgerpc.conf", "/etc/bibos/bibos.conf"]
+DEFAULT_CONFIG_FILES = ["/etc/os2borgerpc/os2borgerpc.conf"]
```

Comment:
I approve of removing the BibOS branding, but isn't this particular change dangerous? What will happen if we upgrade a system with `/etc/bibos/bibos.conf` to a version of the client package that no longer reads that file?

Reply:
The we must move it.

Reply:
No, you're actually right. I'll keep the `bibos` location for backwards compatibility.